src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -1127,6 +1162,7 @@ private class FactoryContext
             public IServiceProviderIsService? ServiceProviderIsService { get; init; }
             public List<string>? RouteParameters { get; init; }
             public bool ThrowOnBadRequest { get; init; }
+            public bool AllowImplicitFromBody { get; init; }
```

Comment:
```suggestion
            public bool AllowInferredFromBody { get; init; }
```

To match the public type name...

Reply:
Yep, I think I'll switch it to "DisableInferredBody"